syntax = "proto2";
import "game_event.proto";

message Event_Shuffle {
    extend GameEvent {
        optional Event_Shuffle ext = 2007;
    }
    optional string zone_name = 1;
}
